pkg/render/common/networkpolicy/networkpolicy.go

```diff
@@ -177,21 +181,51 @@ var KubeAPIServerEntityRule = v3.EntityRule{
 	Selector:          "(provider == 'kubernetes' && component == 'apiserver' && endpoints.projectcalico.org/serviceName == 'kubernetes')",
 	Ports:             Ports(443, 6443, 12388),
 }
+
 var KubeAPIServerServiceSelectorEntityRule = v3.EntityRule{
 	Services: &v3.ServiceMatch{
 		Namespace: "default",
 		Name:      "kubernetes",
 	},
 }
 
-// The entity rules below are extracted from render subpackages to prevent cyclic dependencies.
-var ESGatewayEntityRule = CreateEntityRule("tigera-elasticsearch", "tigera-secure-es-gateway", 5554)
-var ESGatewaySourceEntityRule = CreateSourceEntityRule("tigera-elasticsearch", "tigera-secure-es-gateway")
-var ESGatewayServiceSelectorEntityRule = CreateServiceSelectorEntityRule("tigera-elasticsearch", "tigera-secure-es-gateway-http")
+func Helper(mt bool, ns string) *NetworkPolicyHelper {
```

Comment:
This is meant to make it easier to write policy rules that work in both single-tenant and multi-tenant setups.

The main difference is that the namespace selector in these rules needs to match either the "default" single-tenant namespace (i.e., how it is today) or the dynamically configured tenant namespace.